index.js

```diff
@@ -67,9 +67,23 @@ module.exports = {
       //
       let funnel = require('broccoli-funnel');
       let writeFile = require('broccoli-file-creator');
+      let replace = require('broccoli-string-replace');
       let mergeTrees = require('broccoli-merge-trees');
       let platform = process.env.EMBER_CLI_ELECTRON_BUILD_PLATFORM || process.platform;
 
+      // Write our <base> tag to index.html so electron-protocol-serve will
+      // serve our resources correctly
+      tree = replace(tree, {
+        files: ['index.html', 'tests/index.html'],
+        pattern: {
+          match: /^(\s*)<head>/m,
+          replacement: [
+            '$1<head>',
+            `$1  <base href="serve://dist">`,
```

Comment:
Er, I mean, I don't know what you're talking about, there are no backticks there (yay rebasing 😉 )